### PR TITLE
Updated supported keptn/kind annotation values

### PR DESCRIPTION
**Pull request by TannerGilbert**
Updated supported `keptn/kind` annotation values as there are multiple new values. We're still in the process of standardizing the supported values. Therefore I'll mark this PR as a draft until the Keptn Ecosystem team has discussed this topic in more detail.

### CODE REVIEW

**Comment by TannerGilbert** [comment]
Related to keptn-contrib/artifacthub#101

**Comment by christian-kreuzberger-dtx** [comment]
FYI, there is no strict validation on `keptn/kind` at the moment. The values in the PR are some values we recommend for standardized filtering.

@johannes-b this looks to be good enough for now, do you have anything to add?

File: docs/keptn_annotations.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ You can provide some extra information about your Keptn integrations by using th
 
 - **keptn/kind** *(string)*
 
-This annotation allows providing the kind of the integration. Supported values: *service* and *sli-provider*. It's possible to add multiple kinds by using a comma separated list, like shown in the example below.
+This annotation allows providing the kind of the integration. Supported values: *webhook*, *testing*, *deployment*, *notification*, *observability* and *sli-provider*. It's possible to add multiple kinds by using a comma separated list, like shown in the example below.
```
Comment (johannes-b) [inline docs/keptn_annotations.md:9]: Can we please extend the list by the value: `remediation` in order to cover the auto-remedation use-cases.

**Comment by tegioz** [comment]
Could you please rebase from master @TannerGilbert? 🙂

**Comment by TannerGilbert** [comment]
@tegioz Done.

**Pull request merged**